src/schema/string.ts

```diff
@@ -49,6 +49,14 @@ const formats: {
       )
     }
   },
+  uuid: (key: string, value: string, options: StringFnOptions) => {
+    if (!require('validator/lib/isUUID')(value)) {
+      throw new Error(
```

Comment:
where does this import come from? We dont have `validator` as a dependency in the project